bears/python/PyLintBear.py

```diff
@@ -23,7 +25,8 @@ class PyLintBear:
     separately.
     """
     LANGUAGES = {'Python', 'Python 2', 'Python 3'}
-    REQUIREMENTS = {PipRequirement('pylint', '1.6')}
+    REQUIREMENTS = {AnyOneOfRequirements([PipRequirement('pylint', '1.6'),
```

Comment:
no the voodoo parser doesnt work like that.  I am not sure what you intend to do.  just use '1.6.0' if you are trying to prevent 1.7...?  and then `pylint~=1.6.0` in the `bear-requirements.txt` (run https://github.com/coala/coala-bears/blob/master/.ci/generate_bear_requirements.py to generate this file)

Reply:
alright :+1:

Reply:
so we just have to prevent 1.8, 1.7.5 is working :+1: